### DISCUSSION TITLE
Mobile - Premature submit on new line

### DISCUSSION THREAD

**Discussion by HriGunov** [danny-avila/LibreChat#4268] [Troubleshooting] [answered]
### What happened?

On a mobile browser.
While writing a message, you cannot go to a new line. Pressing the return button submits the current contents.
This makes creating multi-line queries impossible on mobile.

This problem is not present while editing an existing message.

### Steps to Reproduce

1. Open libre-chat on a mobile device
2. Type some message (if the chat input field is empty the issue is not reproducible)
3. Attempt to go to a new line and write some more
 

### What browsers are you seeing the problem on?

Mobile (iOS)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
This problem has already been accounted for. You need to enable the desired behavior in your settings:

![image](https://github.com/user-attachments/assets/54137d50-e595-4c10-a778-1a4508371a11)
